### PR TITLE
Gerrit code reivew

### PR DESCRIPTION
**Pull request by harikishorekuppa**
Have added the Gerrit code reivew  process

---
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/252)

### CODE REVIEW

File: src/main/markdown/makinggwtbetter.md [hunk 1]
```diff
@@ -538,6 +538,12 @@ Finally, to actually create an issue is slightly more involved, but most of the
 
 If you have any problems you might want to consult the gerrit documentation on [Uploading Changes](https://gerrit-review.googlesource.com/Documentation/user-upload.html) 
 
+### Gerrit code review<a id="gerritcodereview"></a>
+
+Code review is an essential part of contribution workflow :
+
+Once your code is pushed to gerrit, notifications will be triggered to all reviewers,at this stage your code need jenkins precheck as well +2 in Code-Review to get merged with remote repository,If you are new contributor Jenkins precheck triggering will be followed by +2 in Code-Review by reviewers,whereas existing  contributors will be added to Jenkins Trusted group which allows the jenkins precheck without wait for code review.
```
Comment (TDesjardins) [inline src/main/markdown/makinggwtbetter.md]: This sentence/sentences needs some formatting :-) . Sometimes missing sometimes to many spaces and a point at the end instead a comma.

File: src/main/markdown/makinggwtbetter.md [hunk 2]
```diff
@@ -538,6 +538,12 @@ Finally, to actually create an issue is slightly more involved, but most of the
 
 If you have any problems you might want to consult the gerrit documentation on [Uploading Changes](https://gerrit-review.googlesource.com/Documentation/user-upload.html) 
 
+### Gerrit code review<a id="gerritcodereview"></a>
+
+Code review is an essential part of contribution workflow :
```
Comment (tbroyer) [inline src/main/markdown/makinggwtbetter.md]: I love that sentence! :smiley: 

The following paragraph needs some work though. Let me think a bit about it, I'll try to propose a new wording.

Good work on the heading and where to put that section too.

**Comment by zbynek** [comment]
@niloc132 maybe this can be closed since Gerrit is no longer part of the workflow?

**Comment by niloc132** [comment]
Agreed (though Thomas if you happen to have those thoughts as a draft, they would surely be welcomed..).

**Pull request closed without merging**